Agents/AverageSquareMetrePriceAgent/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# define agent logging dependency here as there are issues including it in install_requires in setup.py
```

Comment:
I would recommend pinning the working version of the list of dependencies here to be installed from Dockerfile for production.

Reply:
Requirements files and `setup.py` are designed for different purposes. The former tends to be as specific as possible for reproducibility and production, whereas the latter normally gives a wide range to not limit the choice of packages in development. If you let the pip decide the version of packages, there's a chance it pulls a version that potentially breaks the code. You can read more about these two things here https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/

Reply:
Pinned fox version number in [9a93285](https://github.com/cambridge-cares/TheWorldAvatar/pull/398/commits/9a932853c74dce979f8618b402de1bc46559c8d6)